dependencies.yaml

```diff
@@ -142,9 +173,7 @@ dependencies:
           - cxx-compiler
           - fmt>=11.0.2,<12
           - libcumlprims==25.2.*,>=0.0.0a0
-          - libcuvs==25.2.*,>=0.0.0a0
           - libraft-headers==25.2.*,>=0.0.0a0
```

Comment:
We could move/remove `libcumlprims` and `libraft-headers` into their own `depends_on_X` lists. Or maybe we can remove `libraft-headers` here? We did that in cugraph, iirc. This is not a blocker, I defer to your judgment.